components/script/dom/response.rs

```diff
@@ -271,6 +330,7 @@ impl ResponseMethods for Response {
         if *self.body.borrow() != NetTraitsResponseBody::Empty {
             *new_response.body.borrow_mut() = self.body.borrow().clone();
         }
+        new_response.body_used.set(self.BodyUsed());
```

Comment:
I don't believe this is correct - https://fetch.spec.whatwg.org/#concept-body-clone is written weirdly, but it implies that cloning a response's body does not cause the original body to be consumed if it is not already, and the new response's body should not be marked used.